scripts/src/errors.ts

```diff
@@ -116,3 +166,155 @@ export class ChannelBusyError extends Error implements KinSdkError {
 		super('Cannot acquire a free channel.');
 	}
 }
+
+export class BadRequestError extends HorizonError {
+	readonly type: ErrorType = 'BadRequestError';
+
+	constructor(readonly errorBody: ErrorResponse, readonly title?: string) {
+		super(`Bad Request error`, errorBody, title);
+	}
+}
+
+export class InternalError extends HorizonError {
+	readonly type: ErrorType = 'InternalError';
+
+	constructor(readonly errorBody: any, readonly title?: string) {
```

Comment:
do we need a title? why does the message include a json? can't it simply says "unknown error"?